tutorials/users/catalog/pystac-client.ipynb

```diff
@@ -0,0 +1,1929 @@
+        "cmip6:source_id": "EC-Earth3",
+        "cmip6:source_type": [
+         "AOGCM"
+        ],
+        "cmip6:sub_experiment": "none",
+        "cmip6:sub_experiment_id": "none",
+        "cmip6:table_id": "Amon",
+        "cmip6:tracking_id": "",
+        "cmip6:variable_id": "tas",
+        "cmip6:variant_label": "r150i1p1f1",
+        "cmip6:version": "",
+        "cube:dimensions": {
+         "lat": {
+          "axis": "y",
+          "description": "projection_y_coordinate",
+          "extent": [
+           -89.46282196044922,
+           89.46282196044922
+          ],
+          "type": "spatial"
+         },
+         "lon": {
+          "axis": "x",
+          "description": "projection_x_coordinate",
+          "extent": [
+           0,
+           359.296875
+          ],
+          "type": "spatial"
+         },
+         "time": {
+          "description": "time",
+          "extent": [
+           "1970-01-16T12:00:00Z",
+           "2014-12-16T12:00:00Z"
+          ],
+          "type": "temporal"
+         }
+        },
+        "cube:variables": {
+         "height": {
+          "description": "height",
+          "dimensions": [
+           ""
+          ],
+          "type": "auxiliary",
+          "unit": "m"
+         },
+         "lat_bnds": {
+          "description": "",
+          "dimensions": [
+           "lat",
+           "bnds"
+          ],
+          "type": "data",
+          "unit": ""
+         },
+         "lon_bnds": {
+          "description": "",
+          "dimensions": [
+           "lon",
+           "bnds"
+          ],
+          "type": "data",
+          "unit": ""
+         },
+         "tas": {
+          "description": "Near-Surface Air Temperature",
+          "dimensions": [
+           "time",
+           "lat",
+           "lon"
+          ],
+          "type": "data",
+          "unit": "K"
+         },
+         "time_bnds": {
+          "description": "",
+          "dimensions": [
+           "time",
+           "bnds"
+          ],
+          "type": "data",
+          "unit": ""
+         }
+        },
+        "datetime": null,
+        "end_datetime": "2014-12-16T12:00:00Z",
+        "start_datetime": "1970-01-16T12:00:00Z"
+       },
+       "stac_extensions": [
+        "https://raw.githubusercontent.com/TomAugspurger/cmip6/main/json-schema/schema.json",
+        "https://stac-extensions.github.io/datacube/v2.2.0/schema.json"
+       ],
+       "stac_version": "1.0.0",
+       "type": "Feature"
+      },
+      "text/plain": [
+       "<IPython.core.display.JSON object>"
```

Comment:
This doesn't render nicely in the jupyterbook. Maybe add an image here instead to show what it looks like if we can't get it to render properly.

Reply:
Yup, I know. I was going to ask you folks for suggestions. I'm not sure why the book is not rendering it correctly...

Reply:
Yeah it should just be all javascript so it should work. We can look into this more to try to figure out if there's a way to make it work

Reply:
I tried to solve this last week, but was unable to. Eventually I created an [issue](https://github.com/executablebooks/jupyter-book/issues/2106) for this at the Jupyter-book repo last week. Based on the response, it seems this feature is not yet supported.

Reply:
It sounds like they're at least looking into supporting it though which is good. Let's check back in a bit and see if they can add support for this later on.

Reply:
Yes, in the meantime we can add a screenshot.

Reply:
Done